Directory.Build.targets

```diff
@@ -8,6 +8,11 @@
     -->
     <IsShippingPackage Condition=" '$(IsShippingPackage)' == '' ">false</IsShippingPackage>
 
+    <!--
+      By default, assemblies which are only in the Microsoft.AspNetCore.App shared framework are not available as NuGet packages.
+    -->
+    <IsPackable Condition="'$(IsAspNetCoreApp)' == 'true' AND '$(IsShippingPackage)' != 'true'">false</IsPackable>
```

Comment:
Should be `OR`

Reply:
`AND` is deliberate. This ensures we don't produce packages for shared-framework-only assemblies like Mvc.Core, but still do for shared-framework-and-package assemblies like SignalR

Reply:
Decide to back this out. After closer inspection of the nupkg's produced by the build, we need to keep producing the intermediate nupkg's for now. We can tackle this again when we do #4257